webpack.config.js

```diff
@@ -51,11 +51,9 @@ var webpackConfig = {
   entry: {
     "news-node.behaviors":  path.resolve(srcJS,   "news-node.behaviors.js"),
     "news-node":            path.resolve(srcSass, "components/news-node/index.scss"),
-    "news-card":            path.resolve(srcSass, "components/news-card/index.scss"),
     "news-list-item":       path.resolve(srcSass, "components/news-list-item/index.scss"),
     "news-list.behaviors":  path.resolve(srcJS,   "news-list.behaviors.js"),
-    "newsletter":           path.resolve(srcSass, "components/newsletter/index.scss"),
-    "news-assets":          path.resolve(srcSass, "components/newsletter/index.scss")
```

Comment:
@boznik @jenbreese I removed what looked like files that were not being included anywhere. Please confirm.

Reply:
We aren't using those. Should I resolve?